### PR TITLE
Feature#31 seed for app

### LINKED ISSUE #31: Seeds for application
# REQUIREMENTS
In this issue all seeds for application will be created.

# DESIGN
# Acceptance Criteria
List all required acceptance criteria
- [ ] Admin Seeds
- [ ] Companies Seeds
- [ ] Employees Seeds
- [ ] Attendances Seeds

### PR DESCRIPTION
**Pull request by ramasoftdev**
With this PR The App is gonna have a data base with 3 companies, 3 admins and a maximun of 30 employees between companies and 2 months of dates registers
 
## Description: Why did you change/implement that?
A pupulated database.

### CODE REVIEW

File: reloj-checador/db/seeds.rb [hunk 1]
```diff
@@ -7,7 +7,11 @@
...
-    load "db/seeds/admins/admins.rb"
-  end
+  Attendace.destroy_all
+  Employee.destroy_all
+  Company.destroy_all
+  load "db/seeds/admins/admins.rb"
+  load "db/seeds/companies/companies.rb"
+  load "db/seeds/employees/employees.rb"
+  load "db/seeds/attendaces/attendaces.rb"
 end
```
Comment (YilKanda) [inline reloj-checador/db/seeds.rb]: Add a blank line.

File: reloj-checador/db/seeds/admins/admins.rb [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+  { email: "admin1@email.com", password: "pass12345" },
+  { email: "admin2@email.com", password: "pass12345" },
+  { email: "admin3@email.com", password: "pass12345" },
 ]
 
+p "Creating admins"
+
 admins.each { |admin|
-  puts "Adding #{admin[:name]} #{admin[:last_name]} #{admin[:email]} #{admin[:password]}"
+  # puts "Adding #{admin[:email]} #{admin[:password]}"
```
Comment (YilKanda) [inline reloj-checador/db/seeds/admins/admins.rb]: If you don't need it, you can delete it.

File: reloj-checador/app/views/attendaces/_form.html.erb [hunk 3]
```diff
@@ -17,11 +17,9 @@
...
 <script>
-  let clock = document.getElementById("clock")
-  
+  var clock = document.getElementById("att-new-clock");
   setInterval(() => {
     let date = new Date()
     clock.innerHTML = date.toLocaleTimeString();
-    console.log(date.toLocaleTimeString());
   },1000);
 </script>
```
Comment (YilKanda) [inline reloj-checador/app/views/attendaces/_form.html.erb:25]: One suggestion to keep your code clean is to distribute the elements. In this case, the javascript code in a JS file.

File: reloj-checador/app/views/welcome/index.html.erb [hunk 4]
```diff
@@ -19,6 +19,7 @@
 <script>
     var renderTime = function () {
         var time = new Date();
+        var clock = document.getElementById("clock");
         clock.textContent = `${time.getDate()}-${time.getMonth()+1}-${time.getFullYear()} ${time.getHours()}:${time.getMinutes()}:${time.getSeconds()}`
     };
     renderTime();
```
Comment (YilKanda) [inline reloj-checador/app/views/welcome/index.html.erb:25]: One suggestion to keep your code clean is to distribute the elements. In this case, the javascript code in a JS file.

File: reloj-checador/db/seeds.rb (hunk 1 above)
Comment (ramasoftdev) [reply reloj-checador/db/seeds.rb]: Sugestion taken. Code changed.

File: reloj-checador/db/seeds/admins/admins.rb (hunk 2 above)
Comment (ramasoftdev) [reply reloj-checador/db/seeds/admins/admins.rb]: Sugestion taken. Code changed; line deleted

**Pull request merged**